From 63cdaac6e3406bf13cdc9b4a8fa3530d24fa5933 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Fri, 18 May 2018 16:35:34 -0500
Subject: [PATCH] Makefile: fix PLATFORM variable at install target

The variable PLAT was renamed to PLATFORM, but was not updated in the
install target giving a failure when installing ACRN hypervisor.

Fixes: fc2b6fbe86e8 ("Makefile: keep using 'PLATFORM' variable for existing documentation")

Signed-off-by: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 938c01c..c483aa5 100644
--- a/Makefile
+++ b/Makefile
@@ -34,7 +34,7 @@ clean:
 install: hypervisor-install devicemodel-install tools-install
 
 hypervisor-install:
-	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) PLATFORM=$(PLAT) RELEASE=$(RELEASE) install
+	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) PLATFORM=$(PLATFORM) RELEASE=$(RELEASE) install
 
 devicemodel-install:
 	make -C $(T)/devicemodel DM_OBJDIR=$(DM_OUT) install
-- 
2.17.0

